<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Mesh3DSymbolWidget</class>
 <widget class="QWidget" name="Mesh3DSymbolWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>331</width>
    <height>837</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QScrollArea" name="scrollArea">
     <property name="widgetResizable">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scrollAreaWidgetContents">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>329</width>
        <height>835</height>
       </rect>
      </property>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="1">
        <widget class="QComboBox" name="mComboBoxSymbologyType">
         <item>
          <property name="text">
           <string>Simple Symbol</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Advanced Symbol</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="3" column="0" colspan="2">
        <widget class="Line" name="line">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="baseSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item row="4" column="0" colspan="2">
        <widget class="QGroupBox" name="mGroupBoxSimpleSettings">
         <property name="title">
          <string>Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_4">
          <item row="1" column="0">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Altitude clamping</string>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Height</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0" colspan="2">
           <widget class="QgsPhongMaterialWidget" name="widgetMaterial" native="true">
            <layout class="QGridLayout" name="gridLayout_5"/>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QComboBox" name="cboAltClamping">
            <item>
             <property name="text">
              <string>Absolute</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Relative</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>Terrain</string>
             </property>
            </item>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QgsDoubleSpinBox" name="spinHeight">
            <property name="minimum">
             <double>-99999.000000000000000</double>
            </property>
            <property name="maximum">
             <double>99999.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QgsPropertyOverrideButton" name="btnHeightDD">
            <property name="text">
             <string>â€¦</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QCheckBox" name="chkAddBackFaces">
            <property name="text">
             <string>Add back faces</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="6" column="0">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="0" column="0">
        <widget class="QLabel" name="label_8">
         <property name="text">
          <string>Symbology</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0" colspan="2">
        <widget class="QGroupBox" name="mGroupBoxAdvancedSettings">
         <property name="title">
          <string>Advanced Settings</string>
         </property>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="1" colspan="2">
           <widget class="QgsCollapsibleGroupBox" name="mGroupBoxTrianglesSettings">
            <property name="title">
             <string>Triangles Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_3">
             <item row="1" column="0" colspan="2">
              <widget class="QGroupBox" name="groupBoxWireframe">
               <property name="title">
                <string>Wireframe Line Width and Color</string>
               </property>
               <layout class="QHBoxLayout" name="horizontalLayout">
                <item>
                 <widget class="QDoubleSpinBox" name="spinBoxWireframeLineWidth">
                  <property name="suffix">
                   <string> px</string>
                  </property>
                  <property name="value">
                   <double>1.000000000000000</double>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QgsColorButton" name="colorButtonWireframe">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QCheckBox" name="chkWireframe">
               <property name="text">
                <string>Wireframe</string>
               </property>
              </widget>
             </item>
             <item row="0" column="0">
              <widget class="QCheckBox" name="chkSmoothTriangles">
               <property name="text">
                <string>Smooth Triangles</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="6" column="1" colspan="2">
           <widget class="QgsCollapsibleGroupBox" name="mgroupBoxTextureSettings">
            <property name="title">
             <string>Texture Settings</string>
            </property>
            <layout class="QGridLayout" name="gridLayout_6">
             <item row="0" column="0">
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Texture Type</string>
               </property>
              </widget>
             </item>
             <item row="2" column="0" colspan="2">
              <widget class="QWidget" name="mMeshUniqueColorWidget" native="true">
               <layout class="QHBoxLayout" name="horizontalLayout_3">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>Mesh color</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QgsColorButton" name="mMeshUniqueColorButton">
                  <property name="text">
                   <string>...</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QComboBox" name="mComboBoxTextureType">
               <item>
                <property name="text">
                 <string>Unique Color</string>
                </property>
               </item>
               <item>
                <property name="text">
                 <string>Ramp Color Shader</string>
                </property>
               </item>
              </widget>
             </item>
             <item row="1" column="0" colspan="2">
              <widget class="QGroupBox" name="mGroupBoxColorRampShader">
               <property name="title">
                <string>Color Ramp Shader</string>
               </property>
               <layout class="QGridLayout" name="gridLayout_7">
                <property name="leftMargin">
                 <number>9</number>
                </property>
                <property name="topMargin">
                 <number>9</number>
                </property>
                <property name="rightMargin">
                 <number>9</number>
                </property>
                <property name="bottomMargin">
                 <number>9</number>
                </property>
                <item row="1" column="0" colspan="5">
                 <widget class="Line" name="line_2">
                  <property name="orientation">
                   <enum>Qt::Horizontal</enum>
                  </property>
                 </widget>
                </item>
                <item row="0" column="1">
                 <widget class="QLineEdit" name="mColorRampShaderMinEdit"/>
                </item>
                <item row="0" column="0">
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Min</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="2">
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Max</string>
                  </property>
                 </widget>
                </item>
                <item row="0" column="4">
                 <widget class="QPushButton" name="mColorRampShaderMinMaxReloadButton">
                  <property name="maximumSize">
                   <size>
                    <width>24</width>
                    <height>24</height>
                   </size>
                  </property>
                  <property name="text">
                   <string/>
                  </property>
                  <property name="icon">
                   <iconset resource="../../../images/images.qrc">
                    <normaloff>:/images/themes/default/mActionRefresh.svg</normaloff>:/images/themes/default/mActionRefresh.svg</iconset>
                  </property>
                 </widget>
                </item>
                <item row="2" column="0" colspan="5">
                 <widget class="QgsColorRampShaderWidget" name="mColorRampShaderWidget" native="true">
                  <property name="minimumSize">
                   <size>
                    <width>0</width>
                    <height>0</height>
                   </size>
                  </property>
                 </widget>
                </item>
                <item row="0" column="3">
                 <widget class="QLineEdit" name="mColorRampShaderMaxEdit"/>
                </item>
               </layout>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
          <item row="5" column="1" colspan="2">
           <widget class="QgsCollapsibleGroupBox" name="mGroupBoxVerticaleSettings">
            <property name="title">
             <string>Verticale Settings</string>
            </property>
            <layout class="QFormLayout" name="formLayout">
             <item row="0" column="0">
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Verticale Scale</string>
               </property>
              </widget>
             </item>
             <item row="0" column="1">
              <widget class="QDoubleSpinBox" name="spinBoxVerticaleScale">
               <property name="maximum">
                <double>9999999.990000000223517</double>
               </property>
               <property name="value">
                <double>1.000000000000000</double>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QgsColorRampShaderWidget</class>
   <extends>QWidget</extends>
   <header>raster/qgscolorrampshaderwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsDoubleSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>qgsdoublespinbox.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPropertyOverrideButton</class>
   <extends>QToolButton</extends>
   <header>qgspropertyoverridebutton.h</header>
  </customwidget>
  <customwidget>
   <class>QgsPhongMaterialWidget</class>
   <extends>QWidget</extends>
   <header>qgsphongmaterialwidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsColorButton</class>
   <extends>QToolButton</extends>
   <header>qgscolorbutton.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>QgsCollapsibleGroupBox</class>
   <extends>QGroupBox</extends>
   <header>qgscollapsiblegroupbox.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../images/images.qrc"/>
 </resources>
 <connections/>
</ui>
